content/md/products-super-power/understand-data-quality.md

```diff
@@ -0,0 +1,88 @@
+---
+id: understand-data-quality
+themes: products-super-power, first-steps
+title: Understand **data quality**
+popular: false
+ee-only: true
+related:
+---
+
+# About data quality
+What is this **red, orange or green A, B, C, D, E displayed on my product edit form**? What are the **enrichment** and **consistency** grades?
+
+**This is data quality insights!**
+
+Because rich, coherent and consistent product data is important to you, we introduced the concept of "data quality" in our PIM.
```

Comment:
```suggestion
Because rich, coherent and consistent product data is important for delivering great product experiences, we introduced the concept of "data quality" in our PIM.
```